src/MainPage.js

```diff
@@ -1,43 +1,60 @@
-import Form from './components/Form'
 import styled from 'styled-components/macro'
-import { useEffect } from 'react'
+import { useState, useEffect } from 'react'
+import getGeoData from './services/getGeoData'
 import getRkiData from './services/getRkiData'
 import useUserInput from './hooks/useUserInput'
+import Form from './components/Form'
+
 
 function MainPage() {
     
     const { 
         userInput, 
         setUserInput, 
-        county, 
-        setCounty, 
-        incidence, 
-        setIncidence
+        userPlace, 
+        setUserPlace,
+        coordinates,
+        setCoordinates, 
+        countyData, 
+        setCountyData
     } 
     = useUserInput()
+
+    const [ errorMessage, setErrorMessage ] = useState()
     
-    const countyResult = county.charAt(0).toUpperCase() + county.slice(1)
 
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    useEffect(() => getCountyData(), [county])
+    useEffect(() => {userPlace && getCountyData()}, [userPlace])
+    useEffect(() => {coordinates.longitude && getIncidenceData()}, [coordinates])
+    useEffect(() => setErrorMessage(''), [userPlace,coordinates] )
+
+    console.log(coordinates)
```

Comment:
Are ``console.log(coordinates)``and ``console.log(countyData)`` placeholders for future logics? Can be removed, if not necessary :)